pkgs/development/libraries/gtkimageview/default.nix

```diff
@@ -1,10 +1,11 @@
 { fetchurl, lib, stdenv, pkg-config, gtk2 }:
 
 stdenv.mkDerivation rec {
-  name = "gtkimageview-1.6.4";
+  pname = "gtkimageview";
+  version = "1.6.4";
 
   src = fetchurl {
-    url = "http://trac.bjourne.webfactional.com/chrome/common/releases/${name}.tar.gz";
+    url = "http://trac.bjourne.webfactional.com/chrome/common/releases/${pname}-${version}.tar.gz";
```

Comment:
Does not resolve for me.

Reply:
maybe we could use the link from [arch-linux](https://github.com/archlinux/svntogit-community/blob/packages/gtkimageview/trunk/PKGBUILD#L14)?

Reply:
We can do that if you want.